src/wallet/wallet.h

```diff
@@ -1200,6 +1200,8 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
         pwalletdbEncryption = NULL;
     }
 
+    OrchardWallet& GetOrchardWallet() { return orchardWallet; }
```

Comment:
Do we really want to expose `OrchardWallet` outside of `CWallet`, rather than it being an internal implementation detail?